### ISSUE TITLE
Could not find module ‘Convert’

### ISSUE THREAD

**Issue by erikd** [amosr/limp#1]
Cabal installing this package with ghc-7.10 I get:

```
tests/Main.hs:4:18:
    Could not find module ‘Convert’
    It is a member of the hidden package ‘ghc-7.10.1’.
    Perhaps you need to add ‘ghc’ to the build-depends in your .cabal file.
    Use -v to see a list of the files searched for.

tests/Main.hs:5:18:
    Could not find module ‘Simplify’
    It is a member of the hidden package ‘ghc-7.10.1’.
    Perhaps you need to add ‘ghc’ to the build-depends in your .cabal file.
    Use -v to see a list of the files searched for.

limp-0.3.2.0 failed during the building phase. The exception was:
```

**Comment by erikd** [comment]
WIth ghc-7.10.1:

```
 mkdir /tmp/limp-test
 cd /tmp/limp-test
 cabal sandbox init
 cabal install --enable-tests limp
```

results in:

Preprocessing test suite 'test' for limp-0.3.2.0...

tests/Main.hs:4:18:
    Could not find module ‘Convert’
    It is a member of the hidden package ‘ghc-7.10.1’.
    Perhaps you need to add ‘ghc’ to the build-depends in your .cabal file.
    Use -v to see a list of the files searched for.

```
tests/Main.hs:5:18:
    Could not find module ‘Simplify’
    It is a member of the hidden package ‘ghc-7.10.1’.
    Perhaps you need to add ‘ghc’ to the build-depends in your .cabal file.
    Use -v to see a list of the files searched for.
 Updating documentation index
/tmp/limp-test/.cabal-sandbox/share/doc/x86_64-linux-ghc-7.10.1/index.html
cabal: Error: some packages failed to install:
limp-0.3.2.0 failed during the building phase. The exception was:
 ExitFailure 1
```

**Comment by erikd** [comment]
Fixed in the 0.3.2.1 release.
